packages/loader-react/README.md

```diff
@@ -29,23 +29,18 @@ Komponenten tar følgende props:
 -   `negative`: Angir om loaderen skal brukes på mørk bakgrunn. `boolean`
 -   `inline`: Angir om loaderen skal brukes inne i et annet element. `boolean`
 -   `className`: Eventuell(e) css-klassenavn for komponenten. `string`
+-   `politness`: Sett politeness setting på aria-live. `polite` eller `assertive`.
```

Comment:
Kanskje nevne at `polite` er default? Også stavefeil i `politeness` 😊

Reply:
Takk, gikk visst for fort der